README.md

```diff
@@ -1,143 +1,431 @@
+(The negative sign is why the entropy of the 1-choice probability vector `[1.0]` is `-0.0`, as shown above.)
+
+It measures how the probability is dispersed among the possible scores.
+The best case is for each of the `n` possible scores to have probability `1/n` of occurring.In that case, whichever score is returned, there will only be a small number of targets with that score.
+It is not possible to get equal from a starting guess but, sometimes when the target pool is small, a particular guess may be able to split the remaining `k` targets into `k` distinct scores.
+
+In particular, this always occurs when there are only two targets left.
+
+Chosing a guess so as to maximize the entropy provides slightly better performance, on average.
+
+```jl
+julia> wordle2 = GamePool(collect(readlines("./data/Wordletargets.txt")); guesstype=:entropy);
+
+julia> ngwrdl2 = [length(playgame!(wordle2, k).guesses) for k in axes(wordle2.active, 1)];
+
+julia> barplot(countmap(ngwrdl2))
+     ┌                                        ┐ 
+   1 ┤ 1                                        
+   2 ┤■■■■■ 131                                 
+   3 ┤■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■ 978   
+   4 ┤■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■ 928     
+   5 ┤■■■■■■■■ 217                              
+   6 ┤■■ 49                                     
+   7 ┤ 9                                        
+   8 ┤ 2                                        
+     └                                        ┘ 
+
+julia> (mean(ngwrdl2), std(ngwrdl2))
+(3.614254859611231, 0.8552369532287724)
+```
+
+Two of the games that took 8 guesses under the expected pool size strategy also take 8 guesses under the maximum entropy strategy
+
+```jl
+julia> showgame!.(Ref(wordle2), findall(==(8), ngwrdl2));
+┌────────┬────────────┬──────────┬──────────┬──────────┐
+│  guess │      score │ poolsize │ expected │  entropy │
+│ String │     String │    Int64 │  Float32 │  Float32 │
+├────────┼────────────┼──────────┼──────────┼──────────┤
+│  raise │ 🟨🟫🟫🟫🟨 │     2315 │  61.0009 │  5.87791 │
+│  outer │ 🟨🟫🟫🟩🟩 │      102 │  8.68627 │  4.09399 │
+│  mower │ 🟫🟩🟫🟩🟩 │       16 │    5.875 │  1.91974 │
+│  cover │ 🟫🟩🟫🟩🟩 │        9 │  3.44444 │  1.65774 │
+│  joker │ 🟫🟩🟫🟩🟩 │        5 │      2.2 │  1.37095 │
+│  boxer │ 🟫🟩🟫🟩🟩 │        3 │  1.66667 │ 0.918296 │
+│  foyer │ 🟫🟩🟫🟩🟩 │        2 │      1.0 │      1.0 │
+│  goner │ 🟩🟩🟩🟩🟩 │        1 │      1.0 │     -0.0 │
+└────────┴────────────┴──────────┴──────────┴──────────┘
+┌────────┬────────────┬──────────┬──────────┬──────────┐
+│  guess │      score │ poolsize │ expected │  entropy │
+│ String │     String │    Int64 │  Float32 │  Float32 │
+├────────┼────────────┼──────────┼──────────┼──────────┤
+│  raise │ 🟫🟩🟫🟫🟫 │     2315 │  61.0009 │  5.87791 │
+│  tangy │ 🟨🟩🟫🟫🟫 │       91 │  7.48352 │  4.03061 │
+│  caput │ 🟨🟩🟫🟫🟨 │       13 │  2.84615 │   2.4997 │
+│  batch │ 🟫🟩🟩🟩🟩 │        5 │      3.4 │ 0.721928 │
```

Comment:
both of these failures have a weird thing where the entropy drops below 1 bit -- which should be a single guess, before increasing again.

Maybe that's something to note: entropy is dependent on your probability model, which in this case is whether you're looking at the probabilities of score congruence classes or the probabilities of words. Your entropy is on the congruence classes, which means that there might be multiple guesses remaining if they're all in the same class (e.g cover, joker, boxer, foyer are all 02022 here).

I'm not sure my second paragraph makes sense. :thinking: